### PR TITLE
refactor(nodebuilder): Logger names in subpackages to `constructor/<subpkg>`

### LINKED ISSUE #1288: chore(nodebuilder): change `log` loggers to `$MODULENAME/constructor` instead of `$MODULENAME-module`
Self explanatory. Probably should be done after #1176

### PR DESCRIPTION
**Pull request by renaynay**
Resolves #1288

### CODE REVIEW

File: nodebuilder/fraud/module.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/node"
 )
 
-var log = logging.Logger("fraud-module")
+var log = logging.Logger("constructor/fraud")
```
Comment (Wondertan) [inline nodebuilder/fraud/module.go:11]: The constructor is confusing. 
1. Constructor of the fraud references the fraud pkg more
2. This pkg could log out things beyond the module construction.

Let's do `modfraud` or `fraudmod` pls

Comment (distractedm1nd) [reply nodebuilder/fraud/module.go:11]: So, the reasoning behind this is that the user won't know what a "nodebuilder" is nor will understand what the difference between "fraud" and "modfraud" packages would be. I like the constructor pattern but am open for more ideas. Not `modX` pilled yet, do you have any other ideas?

**Review by distractedm1nd — DISMISSED** [review]

File: nodebuilder/fraud/module.go (hunk 1 above)
Comment (Wondertan) [reply nodebuilder/fraud/module.go:11]: The point is that the logger name should reference the fact that it is a module. The `constructor` does not do that.

Comment (Wondertan) [reply nodebuilder/fraud/module.go:11]: Why not just `modfraud` or `mod/fraud`?

**Review by Wondertan — CHANGES_REQUESTED** [review]
The logger is for the module, not only the constructor.
`modfraud` or `mod/fraud` or `module/fraud`

Related #1323

**Review by Wondertan — APPROVED** [review]
Thanks for changing this to module!

**Review by distractedm1nd — APPROVED** [review]
module/x much better than modX, thanks

**Review by vgonkivs — APPROVED** [review]

**Pull request merged**